util/shop/shop.js

```diff
@@ -51,9 +51,9 @@ const sneak = (callback) => {
   callback();
 };
 
-const deposit = (userId, callback, quantity) => {
-  data.addCrowns(userId, quantity ?? 1);
-  if (callback) callback(`💳 You have purchased a Crown Gift Card! (+${quantity ?? 1} 👑)`);
+const deposit = (userId, callback, quantity = 1) => {
+  data.addCrowns(userId, quantity);
+  if (callback) callback(`💳 You have purchased ${quantity} Crown Gift Card${quantity > 1 ? 's' : ''}! (+${quantity} 👑)`);
```

Comment:
Could you actually keep this how it was before? I thought about doing this and decided to keep it since gift cards can have different value!

Reply:
Ohhhhh, true!